tools/validators/instance_validator/validate/telemetry_validator.py

```diff
@@ -93,36 +93,39 @@ def ValidateMessage(self, message):
     self.validated_entities[entity_name] = True
 
     entity = self.entities[entity_name]
-    for point_name, point_config in entity[TRANSLATION].items():
-      if point_name not in t.points.keys():
-        self.AddError(
+    print("Validating entity: ", entity_name)
+    if TRANSLATION in entity.keys():
```

Comment:
Instead of indenting everything under this check, you can invert it and short-circuit.